e7-damage-calc/artifacts.js

```diff
@@ -155,6 +156,12 @@ const artifacts = {
     exclusive: classType.warrior,
     applies: (skill) => skill.single === true && elements.target_has_debuff.value() !== false,
   },
+  junkyard_dog:{
+    id: 'junkyard_dog',
+    name: 'Junkyard Dog',
+    type: artifactDmgType.burn,
```

Comment:
so this would become
```suggestion
    type: artifactDmgType.dot,
    dot: [dot.burn],
```